<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
  <meta name="description" content="Personal blog">
  <meta name="author" content="Sebastien Grec">
  <link rel="icon" href="./image/Arabesc_logo.png">

  <title>Sebastien Grec</title>

  <!-- Bootstrap core CSS -->
  <link href="./dist/css/bootstrap.min.css" rel="stylesheet">

  <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
  <link href="./assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">

  <!-- Custom styles for this template -->
  <link href="blog.css" rel="stylesheet">

  <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
  <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
  <script src="./assets/js/ie-emulation-modes-warning.js"></script>
  <script src="https://embedr.flickr.com/assets/client-code.js"></script>
  <link rel="stylesheet" href="./dist/css/bootstrap.min.css">
  <script src="./dist/js/jquery.min.js"></script>
  <script src="./dist/js/bootstrap.min.js"></script>
  <!-- Custom styles for this template -->
  <!-- <link href="css/navbar-fixed-top.css" rel="stylesheet"> -->
  <link href="./dist/css/sticky-footer-navbar.css" rel="stylesheet">
  <link href="./dist/css/custom-tricks.css" rel="stylesheet">
  <link href="./dist/font-awesome-4.6.3/css/font-awesome.min.css" rel="stylesheet">
  <link href="./dist/academicons-1.7.0/css/academicons.min.css" rel="stylesheet">
  <link href="./dist/flag-icon-css/css/flag-icon.min.css" rel="stylesheet">

  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
  <![endif]-->
</head>

<body>

  <div class="blog-masthead">
    <div class="container">
      <nav class="blog-nav">
        <a class="blog-nav-brand" href="./index.html">
        <a <href="./index.html"> &nbsp; <font size="+3" color="#00000"> Sebastien GREC</font></a>
        <a class="blog-nav-item" href="./index.html"><font size="+2">Home</font></a>
        <!-- <a class="blog-nav-item" href="./new_features.html">New features</a> -->
        <a class="blog-nav-item" href="./about_me.html"><font size="+2">About me</font></a>
        <a class="blog-nav-item active" href="./research.html"><font size="+2">Research</font></a>
        <a class="blog-nav-item" href="./teaching.html"><font size="+2">Teaching</font></a>
        <a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</a>
        <a href="https://www.mendeley.com/profiles/sebastien-grec2/" target="_blank" alt=""><i class="ai ai-lg ai-mendeley"></i></a>
        <a href="https://www.researchgate.net/profile/Sebastien_Grec" target="_blank" alt=""><i class="ai ai-lg ai-researchgate"></i></a>
        <a href="https://scholar.google.fr/citations?view_op=list_works&hl=fr&user=-jrxHFEAAAAJ&gmla=AJsN-F6rQxPFSb5jWrIkTCi-x4TGNoP22TF8kl270H5uPhlpw2vIdQEsjxioViwDjkkiO4iEU08yCiNXpEhvILuTsyHLSlQfouzPPDO0KyXSGhVAdynIbdYViFmgFlzwfuX-eSO-l8SqwEIh7l3FsMgJTalOK5VxQg" target="_blank" alt=""><i class="ai ai-lg ai-google-scholar"></i></a>
        <a href="https://github.com/sebgrec" target="_blank" alt=""><i class="fa fa-lg fa-github"></i></a></li>
        <a href="https://twitter.com/sebgrec" target="_blank" alt=""><i class="fa fa-lg fa-twitter"></i></a></li>
        <a href="https://fr.linkedin.com/in/sebastien-grec-4b558a145" target="_blank" alt=""><i class="fa fa-lg fa-linkedin"></i></a></li>

      </nav>
    </div>
  </div>

<!--début contenu du site---------------------------------------------------------------------------------------------->
    <div class="container">
      <div class="blog-header">
        <p class="lead blog-description"><font size="+4">Plant Cell Wall Glycobiology</font></p>
      </div>
      <div class="row">
        <div class="col-sm-8 blog-main">
          <!--début trio

<!-début contenu du site---------------------------------------------------------------------------------------------->
<div class="blog-post">
  <!--début trio d'images en rond---------------------------------------------------------------------------------------------->
<ul>
  <div class="col-sm-6 col-md-4">
    <div class="img-circle">
      <img src="./image/flaxtronic.jpeg" class="img-circle" alt="...">
      <div class="caption">
        <strong></strong>
      </div>
    </div>
  </div>
  <div class="col-sm-6 col-md-4">
    <div class="img-circle">
      <img src="./image/chercheur_lego.jpeg" class="img-circle" alt="...">
      <div class="caption">
        <strong></strong>
      </div>
    </div>
  </div>
  <div class="col-sm-6 col-md-4">
    <div class="img-circle">
      <img src="./image/racine_cyto.jpeg" class="img-circle" alt="...">
      <div class="caption">
        ©sebgrec
      </div>
    </div>
  </div>
</ul>
  <hr></hr>
  <hr></hr>
</div>
  <!--fin trio d'images en rond---------------------------------------------------------------------------------------------->
<div class="blog-post">
  <h2 class="blog-post-title">"Plant cell wall degradation"</h2>
  <!-- <p class="blog-post-meta">January 1, 2014 by <a href="#">Mark</a></p> -->
  <p class="text-justify">The degradation of the plant wall is a fundamental process allowing the recycling of the biomass produced at the level of the biosphere.
    It's also an important step into transform of biomass into valuable biopolymers, materials or biofuel. In nature, these processes relie on microbial facilities to secrete cell wall degrading enzymes.
    Indeed, pathogenic or saprophytic microorganisms possess a large set of secreted enzymes that are dedicated to depolymerization of the main structural polysaccharide components of the plant cell wall, i.e., cellulose, hemicellulose, and pectin.
    Our laboratory studies the enzymatic functions dedicated to the degradation of polysaccharide polymers of the cell wall.
  </p>
  <p class="text-justify"> For that purpose, we coupled metaOmics  (metabarcoding, metatranscriptomic and metaproteomics) approach,
    to classical enzymology and "wet" plant cell wall polysaccharide chemistry to identify some potential major enzymatic functions related to bacterial and fungal cell wall degradation.
  Classically, this type of approach is carried out on forest beds for which a lot of data already exists.
  Our originality is that we are interested in a model of controlled natural degradation used in the production of natural fibers: the flax dew retting.
</p>
  <h3>Flax dew retting as a model</h3>
  <p class="text-justify">Flax dew retting is the first step in the industrial transformation of flax stems into valuable fibers.
    The flax stems are ‘pulled’ (up-rooted) and laid down directly on the soil in swaths (strips). Subsequent exposure to alternating periods of rain and heat combined with the development and the action of soil microflora processes the separation of cellulosic bast fibres from the stems via the partial degradation of cell wall polymers.
    Despite many studies of this process relatively little is known about : <br>
    i) the composition and the evolution of the microflora population,<br>
    ii) the kinetics of microbial colonization of plant material,<br>
    iii) the composition and the evolution of the plant cell wall-degrading enzymes targeted to the different CAZy families. </p>

    <div class="col-sm-6 col-md-4">
      <div class="thumbnail">
        <img src="./image/flax_bast_fibers.png" alt="...">
        <div class="caption">
          <strong>Flax Stem</strong>
        </div>
      </div>
    </div>
    <div class="col-sm-6 col-md-4">
      <div class="thumbnail">
        <img src="./image/field retting.png" alt="...">
        <div class="caption">
          <strong>Dew Retting</strong>
        </div>
      </div>
    </div>
    <div class="col-sm-6 col-md-4">
      <div class="thumbnail">
        <img src="./image/flax_fibers_zoom.png" alt="...">
        <div class="caption">
          <strong>Flax fiber</strong>
        </div>
      </div>
    </div>
    <h3>MetaOmics</h3>
    <p class="text-justify">Thanks to the metaOmics approaches we were able to establish the first comprehensive microbiota of flax dew retting and identify some potential major enzymatic functions related to bacterial and fungal cell wall degradation (<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2017.02052/full">Djemiel <i>et al</i>, 2017</a>).
      These approaches based on Next Generation of Sequencing (Illumina Hiseq and MiSeq system) can access the exogenous (soil) and endogenous (plant) enzymatic arsenal potentially involved in degrading cell wall polysaccharides
      during dew-retting in flax (<a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.581664/full">Djemiel <i>et al</i>, 2020</a>, <a href="https://www.sciencedirect.com/science/article/abs/pii/S0926669020301710?via%3Dihub">Chabbert <i>et al</i>, 2020</a>).</p>
    <p class="text-justify">Metaomics represent the high-throughput, global analysis of DNA, RNA, proteins, or metabolites isolated directly from a community of organisms living in a particular environment (Human Genome Project Information Archive Glossary, 2012).</p>
      <p class="text-justify"> Metagenomics is the study of genetic material recovered directly from environmental samples. The broad field may also be referred to as environmental genomics, ecogenomics or community genomics. While traditional microbiology and microbial genome sequencing and genomics rely upon cultivated clonal cultures, early environmental gene sequencing cloned specific genes (often the 16S rRNA gene) to produce a profile of diversity in a natural sample.
        Metagenomics allows researchers to access the functional and metabolic diversity of microbial communities, but it cannot show which of these processes are active. The extraction and analysis of metagenomic mRNA (the metatranscriptome) provides information on the regulation and expression profiles of complex communities (Wikipedia 2016).</p>
  <hr></hr>
    <p>&nbsp;</p>
    <div id="myCarousel" class="carousel slide" data-ride="carousel">
      <!-- Indicators -->
      <ol class="carousel-indicators">
        <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
        <li data-target="#myCarousel" data-slide-to="1"></li>
        <li data-target="#myCarousel" data-slide-to="2"></li>
      </ol>
      <!-- Wrapper for slides -->
      <div class="carousel-inner">
        <div class="item active">
          <img src="./image/retting_env.png" alt="Los Angeles" style="width:100%;">
          <div class="carousel-caption">
            <h3> </h3>
            <p> </p>
          </div>
        </div>
        <div class="item">
          <img src="./image/retting_cell.png" alt="Chicago" style="width:100%;">
          <div class="carousel-caption">
            <h3> </h3>
            <p> </p>
          </div>
        </div>

        <div class="item">
          <img src="./image/retting_enz.png" alt="New york" style="width:100%;">
          <div class="carousel-caption">
            <h3> </h3>
            <p> </p>
          </div>
        </div>
      </div>
      <!-- Left and right controls -->
      <a class="left carousel-control" href="#myCarousel" data-slide="prev">
        <span class="glyphicon glyphicon-chevron-left"></span>
        <span class="sr-only">Previous</span>
      </a>
      <a class="right carousel-control" href="#myCarousel" data-slide="next">
        <span class="glyphicon glyphicon-chevron-right"></span>
        <span class="sr-only">Next</span>
      </a>
    </div>
    <hr></hr>
</div><!-- /.blog-post -->
<div class="blog-post">
  <h2 class="blog-post-title">TiLLING approach to study cell wall construction</h2>
  <!-- <p class="blog-post-meta">January 1, 2014 by <a href="#">Mark</a></p> -->
  <h3>Project description</h3>
  <p class="text-justify">The flax genome has been sequenced and transcriptomics are providing information on candidate genes potentially related to agronomically-important traits.
  In order to accelerate functional characterization of these genes we have generated a flax EMS mutant population that can be used as a TILLinG (Targeting Induced Local Lesions in Genomes)
   platform for forward and reverse genetics.</p>
  <h3>Main results</h3>
   <p class="text-justify">We have developed a more than 4000 flax mutants population that can be used as an efficient forward and reverse genetics tool. The collection has an extremely high mutation rate that enables the detection of large numbers of independant mutant families by screening a comparatively low number of M2 families.
     The population will prove to be a valuable resource for both fundamental research and the identification of agronomically-important genes for crop improvement in flax <a href="https://bmcplantbiol.biomedcentral.com/articles/10.1186/1471-2229-13-159">(Chantreau <i> et al</i>, 2013)</a>.
     Moreover, identified mutants producing modified fibers composition leads to a better understand of how fiber are constructed inside flax stem <a href="https://doi.org/10.1105/tpc.114.130443">(Chantreau <i> et al</i>, 2014)</a>.</p>

     <div class="col-sm-6 col-md-4">
       <div class="thumbnail">
         <img src="./image/process_tilling.png" alt="...">
         <div class="caption">
           <strong>Mutagenesis procedure</strong>
           ©Grec et al, 2019
         </div>
       </div>
     </div>
     <div class="col-sm-6 col-md-4">
       <div class="thumbnail">
         <img src="./image/mutants.jpg" alt="...">
         <div class="caption">
           <strong>Mutants' phenotypes</strong>
           ©Chantreau et al, 2013
         </div>
       </div>
     </div>
     <div class="col-sm-6 col-md-4">
       <div class="thumbnail">
         <img src="./image/screening.png" alt="...">
         <div class="caption">
           <strong>Mutants selection strategy</strong>
           ©Grec et al, 2019
         </div>
       </div>
     </div>
     <h3>&nbsp;</h3>
</div><!-- /.blog-post -->
<!--Fin contenu du site------------------------------------------------------------------------------------------------>
          <nav>
            <ul class="pager">
              <li><a href="./about_me.html">Previous</a></li>
              <li><a href="./teaching.html">Next</a></li>
            </ul>
          </nav>
<!--Flèche navigation-------------------------------------------------------------------------------------------------->

<!--Debut de la sidebar------------------------------------------------------------------------------------------------>

</div><!-- /.blog-main -->
<div class="sidebar-module">
<div class="col-sm-3 col-sm-offset-1 blog-sidebar">
<div class="sidebar-module">
  <img src="./image/sebgrec_field.jpg" alt="sebgrec_photo" class="center-block img-rounded img-responsive">
<br>
  <!-- <a href="#" class="thumbnail">
    <img src="./image/seb_tarriere.jpg" alt="...">
  </a> -->
</div>
<div class="sidebar-module sidebar-module-inset">
  <h4>Sebastien Grec</h4>
  <table>
    <tr>
      <tr>
        <td><span class="fa fa-graduation-cap" aria-hidden="true">&nbsp;&nbsp;</span></td>
        <td>PhD in Plant Sciences.</td>
      </tr>
      <tr>
        <td></td>
        <td>Engineer in agronomical sciences</td>
      </tr>
  </table>
  <table>
    <tr>
      <td><span class="fa fa-university" aria-hidden="true">&nbsp;&nbsp;</span></td>
      <td>Professor-assistant</td>
    </tr>
    <tr>
      <td></td>
      <td>University of Lille</td>
    </tr>
    <tr>
      <td></td>
      <td>Faculty of Sciences and Technology</td>
    </tr>
  </table>
  <table>
    <tr>
      <td><span class="fa fa-briefcase" aria-hidden="true">&nbsp;&nbsp;</span></td>
      <td>Biology|bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching|Innovating</td>
    </tr>
  </table>
  <table>
    <tr>
      <td><span class="fa fa-envelope" aria-hidden="true">&nbsp;&nbsp;</span></td>
      <td>first.last@univ-lille.fr</td>
    </tr>
  </table>
  <table>
    <tr>
      <td><span class="fa fa-map-marker" aria-hidden="true">&nbsp;&nbsp;&nbsp;</span></td>
      <td>Lille, France</td>
    </tr>
  </table>
</div>
<!-- <div class="sidebar-module">
  <h4>New post</h4>
  <ol class="list-unstyled">
    <iframe src="./sidebar_news.html"width="200" height="210">
    </iframe>
  </ol>
  </div> -->
  <div class="sidebar-module">
    <h4>Follow on Twitter</h4>
    <a class="twitter-timeline" data-width="400" data-height="500" href="https://twitter.com/sebgrec">Tweets by sebgrec</a> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
</div>
</div><!-- /.blog-sidebar -->

</div><!-- /.row -->

</div><!-- /.container -->

</div>

<!--Début footer-------------------------------------------------------------------------------------------------->

<div class="footer">
<div class="container">
<p class="text-muted"><i class="fa fa-copyright fa-fw" aria-hidden="true"></i>Powered by <a href="http://getbootstrap.com/" target="_blank" class="text-orange"> bootstrap v3.3.6</a> | Sebastien Grec | 2022 | Last updated: 08.01.2022</p>
</div>
</div>
<!--Script Java-------------------------------------------------------------------------------------------------------->
<script> window.twttr = (function(d, s, id) {
var js, fjs = d.getElementsByTagName(s)[0],
t = window.twttr || {};
if (d.getElementById(id)) return t;
js = d.createElement(s);
js.id = id;
js.src = "https://platform.twitter.com/widgets.js";
fjs.parentNode.insertBefore(js, fjs);
t._e = [];
t.ready = function(f) {
t._e.push(f);
};
return t;
}(document, "script", "twitter-wjs"));</script>
